integration-tests/openapi-java/src/main/java/org/apache/camel/quarkus/component/openapijava/it/OpenApiRoutes.java

```diff
@@ -129,15 +144,34 @@ public void configure() throws Exception {
                 .withScope("scope2", "Scope 2")
                 .withScope("scope3", "Scope 3")
                 .end()
-                .end()
-
-                .get("/security/openid")
-                .route()
-                .setBody().constant("/security/openid")
-                .endRest()
-                .securityDefinitions()
-                .openIdConnect("openId", "https://secure.apache.org/fake/openid-configuration")
                 .end();
+
+        if (openApiVersion.equals("3.0.0")) {
```

Comment:
Would it perhaps be worth to make this forward-compatible with versions newer than 3.0.0?

Reply:
TBH - the minor & patch parts of the version string are a bit of an irrelevance. There's some logic in camel to check the major version prefix and that's how it decides whether to produce output for OpenApi V2 or V3.